gslib/addlhelp/shim.py

```diff
@@ -96,6 +106,12 @@
     are ignored for cloud objects. This is fixed for the shim and gcloud
     storage.
 
+  iam
```

Comment:
See the design doc's second item in the caveats section (not posting links externally)

Reply:
I just added a note about it.